README.md

```diff
@@ -1,3 +1,9 @@
+<p style="text-align:center;" align="center"><a href="https://layer5.io/meshery"><picture align="center">
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/meshery/meshery-logo-dark-text-side.svg"  width="70%" align="center" style="margin-bottom:20px;">
```

Comment:
please do not use meshery/meshery 's logo, store it locally( if meshsync doesn't have it ) and then use relative referencing to get it.